eventLab.js

```diff
@@ -0,0 +1,23 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let form = document.querySelector("form");
+  form.addEventListener("submit", e => {
+    e.preventDefault();
+    let li = document.createElement("li");
+    let task = document.querySelector("#task");
+    let ul = document.querySelector("#list");
+    let newTask = task.value;
+    let p = document.createElement("p");
+    if (newTask === "") {
+      p.innerText = "Error no input";
+      document.body.appendChild(p);
+    } else {
+      li.innerText = newTask;
+      ul.appendChild(li);
+    }
+    ul.addEventListener("click", e => {
+      let strike = document.createElement("strike");
```

Comment:
the `strike` tag has been deprecated , which means its not supported. Find a way to accomplish this with some CSS. You can do it!